server.js

```diff
@@ -1,9 +1,20 @@
 const express = require('express');
+const mongoose = require('mongoose');
+const config = {
+	mongoURI:
+		'mongodb+srv://akshitp:E7lFFxThDg8fe7Dd@cluster0.6uq2j9y.mongodb.net/?retryWrites=true&w=majority'
+}
 
 const app = express();
 
 app.get('/ping', (req, res) => {
-    res.json({'message': 'pong'})
+    mongoose.connect(config.mongoURI)
+    .then(()=> {
+        res.send("Connection is done.")
+    })
+    .catch(() => {
```

Comment:
Handle the error in the catch block. Currently, the error is being swallowed and not logged anywhere. It would be beneficial to log the error for debugging purposes. [important]